src/Discovery/DiscoveryDetails.tsx

```diff
@@ -0,0 +1,177 @@
+          let groupWidth;
+          switch (fieldGroup.groupWidth) {
+          case 'full':
+            groupWidth = 'fullwidth';
+            break;
+          case 'half':
+          default:
+            groupWidth = 'halfwidth';
+            break;
+          }
+          return (
+            <div key={i} className={`discovery-modal__attribute-group discovery-modal__attribute-group--${groupWidth}`}>
+              { fieldGroup.includeName
+                  && <h3 className='discovery-modal__attribute-group-name'>{fieldGroup.groupName}</h3>}
+              { fieldGroup.fields.map((field) => {
+              // display nothing if selected study doesn't have this field
+              // and this field isn't configured to show a default value
+                if (!props.modalData[field.field] && !field.includeIfNotAvailable) {
+                  return null;
+                }
+                // If the field contains a particularly long string, add some special styles
+                const MULTILINE_FIELD_CHARLIMIT = 200;
+                const multiline = props.modalData[field.field]
+                  && props.modalData[field.field].length > MULTILINE_FIELD_CHARLIMIT;
+                return (
+                  <div key={field.name} className='discovery-modal__attribute'>
+                    { field.includeName !== false
+                        && <span className='discovery-modal__attribute-name'>{field.name}</span>}
+                    <span className={`discovery-modal__attribute-value ${multiline ? 'discovery-modal__attribute-value--multiline' : ''}`}>
+                      { props.modalData[field.field]
+                        ? renderFieldContent(props.modalData[field.field], field.contentType, props.config)
+                        : (field.valueIfNotAvailable || 'Not available')}
+                    </span>
+                  </div>
+                );
+              })}
+            </div>
+          );
+        })}
+      </div>
+      { (props.config.studyPageFields.downloadLinks && props.config.studyPageFields.downloadLinks.field
+        && props.modalData[props.config.studyPageFields.downloadLinks.field])
+        ? (
+          <Collapse className='discovery-modal__download-panel' defaultActiveKey={['1']}>
+            <Panel
+              className='discovery-modal__download-panel-header'
+              header={props.config.studyPageFields.downloadLinks.name || 'Data Download Links'}
+              key='1'
+            >
+              <List
+                itemLayout='horizontal'
+                dataSource={props.modalData[props.config.studyPageFields.downloadLinks.field]}
+                renderItem={(item:ListItem) => (
+                  <List.Item
+                    actions={[
+                      <Button
+                        className='discovery-modal__download-button'
+                        href={`${userAPIPath}/data/download/${item.guid}?expires_in=900&redirect`}
```

Comment:
i think the variable `fenceDownloadPath` from localconf.js would be useful here
https://github.com/uc-cdis/data-portal/blob/eaa0e0c1c148c6f4b65517c589e7d3c7a0443a99/src/localconf.js#L288